trace/kafka/kafka.go

```diff
@@ -98,11 +99,27 @@ func (ap *AsyncProducer) Error() <-chan error {
 	return ap.chErr
 }
 
-// Close gracefully the producer.
+// ActiveBrokers returns a list of active brokers' addresses.
+func (ap *AsyncProducer) ActiveBrokers() []string {
+	brokers := ap.prodClient.Brokers()
+	activeBrokerAddresses := make([]string, len(brokers))
+	for i, b := range brokers {
```

Comment:
Would you consider using „copy“ for a slice ? And let go do the memory re-arrangement under the hood ?

Reply:
@drakos74 can you make an example for this specific loop? I don't think `copy` can be used here.